app/frontend/src/lib/message-service.ts

```diff
@@ -1,25 +1,31 @@
 import Dexie from "dexie";
 import type { EntityTable } from "dexie";
+import type { User } from "@server/db/schema.sql";
 
 export interface Message {
 	id?: string;
 	content: string;
 	timestamp: number;
 	status: "sent" | "received";
+	userId: string;
+	username: string;
```

Comment:
no need for username in the message

Reply:
Renoved username and status, because its not needed anymore